examples/ImageStreamer/create_server_profile_based_on_server_profile_template.yml

```diff
@@ -0,0 +1,44 @@
+###
```

Comment:
This file seems kind of pointless to me, it lists creating a SP based on a template and powering it on, but that is  already illustrated in the basic oneview_server_profile.yml example: [link](https://github.com/HewlettPackard/oneview-ansible/blob/master/examples/oneview_server_profile.yml#L50)

Reply:
yes, but it was part of the requested scenarios, I renamed to step 3 and use it again in step 4, so the user can compare tasks.